retrace/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+cp -R $SRC_DIR/retrace $PREFIX/retrace
```

Comment:
Was wondering how this would be found...but this means it's at `$MEMFAULT_DEPS_ROOT`, which is set by the other memfault-config package. 

Nice!